recipes/mosquitto/mosquitto.inc

```diff
@@ -0,0 +1,62 @@
+DESCRIPTION = "Mosquitto is an open source implementation of a\
+	    server for version 3.1 and 3.1.1 of the MQTT protocol"
+
+HOMEPAGE = "http://mosquitto.org"
+LICENSE = "EPL-1.0"
+
+COMPATIBLE_HOST_ARCHS = ".*linux"
+RECIPE_TYPES = "machine"
+
+SRC_URI = "https://github.com/eclipse/mosquitto/archive/v${PV}.tar.gz"
+
+S = "${SRCDIR}/mosquitto-${PV}"
+
+inherit c++ make auto-package-libs auto-package-utils
+
+RECIPE_FLAGS = "mosquitto_uuid mosquitto_websockets"
+DEFAULT_USE_mosquitto_uuid = "1"
+DEFAULT_USE_mosquitto_websockets = "0"
```

Comment:
Just because that's the out-of-the-box default - the same reason for making uuid default on.